NAMESPACE

```diff
@@ -53,6 +53,9 @@ export(getTwitter)
 export(getTwitterToken)
 export(get_sentiment)
 export(get_stopwords)
+export(get_url_param)
+export(get_url_subdomain)
+export(get_url_suffix)
```

Comment:
I think suffix is good enough. See this doc.
https://cran.r-project.org/web/packages/urltools/urltools.pdf